src/vault/core.clj

```diff
@@ -147,7 +147,9 @@
       Whether or not to renew this secret when the lease is near expiry.
     - `:rotate`
       Whether or not to rotate this secret when the lease is near expiry and
-      cannot be renewed.")
+      cannot be renewed.
+    - `:force-renew`
+      Whether or not to force renew this secret regardless of lease expiry.")
```

Comment:
This could be slightly clearer I think - the fact that the option ends in `-renew` makes it seem related to the `:renew` option, but that causes action in the future while this causes action now. In hindsight, I kind of wish I'd named these `:auto-renew` and `:auto-rotate`.

How about:
> Force the secret to be read from the server even if there is a valid lease cached.

A better name might be `:force-read`?